src/client/utils.js

```diff
@@ -10,14 +8,12 @@ const dispatchEvent = (event) => {
 	document.body.dispatchEvent(event);
 };
 
-const getMessageRules = (messageId) => {
-	return get(manifest, `${messageId}.eventRules`);
-};
+const getMessageRules = messageId => manifest[`${messageId}.eventRules`];
 
 const updateLocalCounter = (messageId, event) => {
 	const currentCounts = cookies.get(LOCAL_COUNTER_COOKIE_NAME) ? JSON.parse(cookies.get(LOCAL_COUNTER_COOKIE_NAME)) : {};
-	const currentCount = get(currentCounts, `${messageId}.${event}`) || 0;
-	set(currentCounts, `${messageId}.${event}`, currentCount + 1);
+	const currentCount = currentCounts[`${messageId}.${event}`] || 0;
```

Comment:
Bracket notation does not support deep property gets such as this. We will need to rewrite this to be:
```suggestion
	const currentCount = currentCounts[messageId] != null && currentCounts[messageId][event] != null ?  currentCounts[messageId][event] :  0;
```

Reply:
thanks, I only just noticed this. Just going to check what events this is dealing with, in case they have a `.` in the event name which is important

Reply:
they do not!